sql/sys_vars.cc

```diff
@@ -5464,15 +5466,19 @@ Sys_var_rpl_filter::global_value_ptr(THD *thd,
 
   return ret;
 }
-
-static Sys_var_rpl_filter Sys_replicate_do_db(
-       "replicate_do_db", OPT_REPLICATE_DO_DB,
+static Sys_var_rpl_filter Sys_replicate_rewrite_db(
+       "replicate_rewrite_db", OPT_REPLICATE_REWRITE_DB,
        "Tell the slave to restrict replication to updates of tables "
        "whose names appear in the comma-separated list. For "
        "statement-based replication, only the default database (that "
        "is, the one selected by USE) is considered, not any explicitly "
        "mentioned tables in the query. For row-based replication, the "
        "actual names of table(s) being updated are checked.",
+       PRIV_SET_SYSTEM_GLOBAL_VAR_REPLICATE_REWRITE_DB);
+
+static Sys_var_rpl_filter Sys_replicate_do_db(
```

Comment:
I think you accidentally switched `Sys_replicate_do_db` with `Sys_replicate_rewrite_db` here